domain/src/main/java/org/oppia/domain/topic/TopicController.kt

```diff
@@ -210,6 +210,13 @@ class TopicController @Inject constructor(
       .addStory(createTestTopic0Story0())
       .addStory(createTestTopic0Story1())
       .addSkill(createTestTopic0Skill0())
+      .addSkill(createTestTopic0Skill0())
+      .addSkill(createTestTopic0Skill0())
+      .addSkill(createTestTopic0Skill0())
```

Comment:
Added for testing

Reply:
@nikitamarysolomanpvt Good change to check the UI fully.